### PR TITLE
Unite components NavBar, Footer and MarketingCard in Home Page

### LINKED ISSUE #17: Acabar página Home con los componentes: NavBar, Footer y MarketingCard

### PR DESCRIPTION
**Pull request by elenamsd**
Fix #17

### CODE REVIEW

**Review by AdanRL — APPROVED** [review]

**Review by AdanRL — APPROVED** [review]

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -1,34 +1,52 @@
+import { CSSProperties, FC } from "react";
+
+import { styled } from "@mui/material/styles";
+
 import { Grid }  from "@mui/material";
 
-export default function Footer() {
+const GridFooterContainer = styled(Grid)(() => ({
+  backgroundColor:"#937A61",
```
Comment (mnzgarcia) [inline src/components/Footer/Footer.tsx:8]:
```suggestion
  backgroundColor: "#937A61",
```

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -1,34 +1,52 @@
...
+  justifyContent: "center"
+}));
+
+const imgLogo: CSSProperties = {
+  height: 40,
+  width: 40
+};
+
+export const Footer: FC = () => {
   return(
```
Comment (mnzgarcia) [inline src/components/Footer/Footer.tsx:19]:
```suggestion
  return (
```

**Review by AdanRL — APPROVED** [review]

File: src/components/Footer/Footer.tsx [hunk 3]
```diff
@@ -1,34 +1,52 @@
+import { CSSProperties, FC } from "react";
+
+import { styled } from "@mui/material/styles";
+
```
Comment (mnzgarcia) [inline src/components/Footer/Footer.tsx:4]:
```suggestion
```

File: src/components/MarketingCard/MarketingCard.tsx [hunk 4]
```diff
@@ -1,5 +1,6 @@
-import { Card, CardContent, CardMedia, Typography, Grid } from "@mui/material";
 import { FC } from "react";
+
+import { Card, CardContent, CardMedia, Typography, Grid } from "@mui/material";
 import {styled} from "@mui/material/styles";
```
Comment (mnzgarcia) [inline src/components/MarketingCard/MarketingCard.tsx:4]:
```suggestion
import { styled } from "@mui/material/styles";
```

File: src/pages/Home/Home.tsx [hunk 5]
```diff
@@ -1,19 +1,99 @@
...
+const ButtonLogIn = styled(Button) (() => ({
+  color: "white",
+}));
+
+const divTopStyle: CSSProperties = {
+  backgroundColor: "rgb(146, 102, 52, 0.7)",
+};
+
+const GridMarketingContainer = styled(Grid)(() => ({
+  backgroundColor:"#f8f5f2",
```
Comment (mnzgarcia) [inline src/pages/Home/Home.tsx:33]:
```suggestion
  backgroundColor: "#f8f5f2",
```

**Review by mnzgarcia — CHANGES_REQUESTED** [review]

File: src/components/Footer/__tests__/Footer.test.tsx [hunk 6]
```diff
@@ -1,8 +1,7 @@
 import { render } from "@testing-library/react";
-import Footer from "../Footer";
+import { Footer } from "../Footer";
```
Comment (mnzgarcia) [inline src/components/Footer/__tests__/Footer.test.tsx:4]:
```suggestion
```

File: src/components/MarketingCard/MarketingCard.tsx [hunk 7]
```diff
@@ -20,32 +21,34 @@ const ContainerInfo = styled(CardContent)(() => ({
...
+}));
+
 export const MarketingCard: FC<Props> = ({ name, cardImage, info, altInfo }) => {
   return (
     <Grid item xs={11} md={11} lg={7}>
-      <Container>
+      <CardContainer>
         <CardMedia
           component="img"
           sx={{ width:{xs: 150, md: 300, lg: 400} }}
```
Comment (mnzgarcia) [inline src/components/MarketingCard/MarketingCard.tsx:43]:
```suggestion
          sx={{ width: {xs: 150, md: 300, lg: 400} }}
```

File: src/components/NavBar/NavBar.tsx [hunk 8]
```diff
@@ -32,15 +34,12 @@ export const NavBar: FC = () => {
       <div style={divTopStyle}>
         <img style={imgLogoStyle} src="/assets/avocado.png" alt="avocado" />
         <h1 style={h1TitleStyle}>Nutriguide</h1>
-        <LogInButton 
+        <ButtonLogIn
```
Comment (mnzgarcia) [inline src/components/NavBar/NavBar.tsx:37]:
```suggestion
        <ButtonLogIn variant="text" endIcon={<AccountCircle />}>
```

File: src/components/NavBar/NavBar.tsx [hunk 9]
```diff
@@ -32,15 +34,12 @@ export const NavBar: FC = () => {
       <div style={divTopStyle}>
         <img style={imgLogoStyle} src="/assets/avocado.png" alt="avocado" />
         <h1 style={h1TitleStyle}>Nutriguide</h1>
-        <LogInButton 
+        <ButtonLogIn      
           variant="text"
```
Comment (mnzgarcia) [inline src/components/NavBar/NavBar.tsx:38]:
```suggestion
```

File: src/components/NavBar/NavBar.tsx [hunk 10]
```diff
@@ -32,15 +34,12 @@ export const NavBar: FC = () => {
       <div style={divTopStyle}>
         <img style={imgLogoStyle} src="/assets/avocado.png" alt="avocado" />
         <h1 style={h1TitleStyle}>Nutriguide</h1>
-        <LogInButton 
+        <ButtonLogIn      
           variant="text" 
           endIcon={<AccountCircle />}
```
Comment (mnzgarcia) [inline src/components/NavBar/NavBar.tsx:39]:
```suggestion
```

File: src/components/NavBar/NavBar.tsx [hunk 11]
```diff
@@ -32,15 +34,12 @@ export const NavBar: FC = () => {
...
-          sx={{
-            padding: "1em"
-          }}
         >
```
Comment (mnzgarcia) [inline src/components/NavBar/NavBar.tsx:40]:
```suggestion
```

**Review by mnzgarcia — CHANGES_REQUESTED** [review]

**Review by mnzgarcia — APPROVED** [review]
👌🏽

**Pull request merged**